web-server/src/controllers/tasks/create.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ id: number;
+  title: string;
+  slug: string;
+  description: string;
+  statement: string;
+  */
+import { Request, Response, NextFunction } from 'express';
+
+import { AppDataSource } from 'orm/data-source';
+import { Task } from 'orm/entities/tasks/Task';
+import { CustomError } from 'utils/response/custom-error/CustomError';
+
+export const create = async (req: Request, res: Response, next: NextFunction) => {
+  const { ownerId, title, slug, description, statement } = req.body;
+
+  const taskRepository = AppDataSource.getRepository(Task);
+  try {
+    const task = await taskRepository.findOne({ where: { slug } });
+
+    if (task) {
+      const customError = new CustomError(400, 'General', 'Task already exists', [`Task ${slug} already exists`]);
+      return next(customError);
+    }
+
+    try {
+      const newTask = new Task();
+      newTask.ownerId = ownerId;
+      newTask.title = title;
+      newTask.slug = slug;
+      newTask.description = description;
+      newTask.statement = statement;
+      await taskRepository.save(newTask);
+
+      res.customSuccess(200, 'Task successfully created.');
```

Comment:
Tradition is to send the `newTask` back as the response. It's also useful because if you create a new task by clicking save, then click save again, the obvious thing to do is to update the last created task. So the page needs to learn the id of the created task for improved UX.